docs/user/alerting/action-types/webhook.asciidoc

```diff
@@ -72,4 +72,16 @@ Password::  An optional password. If set, HTTP basic authentication is used. Cur
 
 Webhook actions have the following properties:
 
-Body::      A json payload sent to the request URL.
+Body::      A JSON payload sent to the request URL. For example: 
++
+[source,text]
+--
+{
+  "short_description": "{{context.rule.name}}",
+  "description": "{{context.rule.description}}",
+  ...
+}
+--
+
+Note that the characters are escaped so they don't contain double quotes, \n characters, and so on.
```

Comment:
It's possible a customer could land on this page and not understand what the `{{ ... }}` parts of this JSON are.  The only doc we have explaining this, is here: https://www.elastic.co/guide/en/kibana/current/defining-alerts.html#defining-alerts-actions-details

So I think this sentence should reference that link.  Something like:

> Note that the text with double braces `{{}}` are mustache template variables provided an alert.  See <link-here> for more information on mustache template variables.

That link also has some detail on the escaping, so that's maybe not needed here.  If we want to keep it, then we can be more precise and say "mustache template variables are escaped so that the final JSON will be valid JSON (eg, escaping double quote characters)"

BTW, I'd guess other action types may want to add similar doc, so this PR may become a template for those other actions.  The only unique part of the description would be the escaping for JSON (no other actions do JSON escaping, some other actions do other types of escaping)

Reply:
incorporated the link and changed it accordingly.